gallery/visualtests/nxButton.js

```diff
@@ -99,9 +99,68 @@ describe('NxButton', function() {
     it('shows tooltips', hoverTest(iconOnlyBtnExample, selector, true));
   });
 
+  describe('Small NxButton section', function() {
```

Comment:
As I suggested on slack at one point, you may want to separate these into a separate test file so that they can be run in parallel with the existing tests.  `nxButton.js` was already the longest-running visual test before these additions, so anything further added to this file is going to have a significant impact on the runtime of the visual tests overall.

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/d1edf6629c3d640d8f16dafce7bcbdacb0194ce0